light_prop/lightfield.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+
+class LightField:
+    def __init__(self, amp: np.array, phase: np.array):
+        if not (isinstance(amp, np.ndarray) and isinstance(phase, np.ndarray)):
+            raise Exception("Arguments must be np.array type")
+
+        if amp.size != phase.size:
+            raise Exception("Dimensions do not match")
+
+        self.amplitude = amp
+        self.phase = phase
+
+    def to_abs(self):
```

Comment:
Again my understanding/convention - `to_something`, `to` implies conversion from `self` to `something`, so actually returning new object. This returns reference to inner data so I'd opt to either `abs` or `get_abs`